CDCgov/NEDSS-Modernization
# Cnft1 1505 clearing search

## Description

When a user would switch between searches from the regular search to event search the results would carry over.

## Tickets
[CNFT1-1505]

* [Jira Ticket](url)
[CNFT1-1505](https://cdc-nbs.atlassian.net/browse/CNFT1-1505)

## Steps to verify

1. Navigate to advanced search
2. search anything that returns results found
3. navigate to event search
4. see results from previous results cleared. 


[CNFT1-1505]: https://cdc-nbs.atlassian.net/browse/CNFT1-1505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1505]: https://cdc-nbs.atlassian.net/browse/CNFT1-1505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
